### ISSUE TITLE
Luminosity ratio of the focus indicator for the table rows with background is less than required ratio 3:1: A11y_WinForm.NETRuntime_Non-TextContrast

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13761]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Keyboard; #A11yMAS

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle.  
2. Splash screen appears and SSMA is launched 
3. Navigate to the 'Tools' menu and select it. 
4. Navigate to 'Project settings menu item and select it. 
5. Navigate to "Type Mapping" button select it. 
6. Navigate all controls under "Type Mapping" button. 
7. Verify whether luminosity ratio for the focus indicator with background is less than required ratio or not. 

### Actual Result:
Luminosity ratio of the focus indicator for the table rows with background is less than required ratio 3:1.

### Expected Result:
Luminosity ratio of the focus indicator for the table rows with background should satisfy the required ratio 3:1.

### User Empact:
Low vision users will be impacted. Luminosity contrast ratio for focus indicator is less than minimum required ratio i.e. 3:1. Due to this issue user will be unable to track the keyboard focus easily. 

Sample App used:-

[[[[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=M4uY2O)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=cCZLff)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=wBZc6R)

### Attachment:

<img width="882" height="481" alt="Image" src="https://github.com/user-attachments/assets/caed7afc-412d-4277-9fb7-a4ef682f977d" />

**Comment by LeafShi1** [comment]
@merriemcgaw  This issue has been fixed in .net 9.0 with PR #10176, do we need to backport this commit to .net 8.0 and other lower version?

<img width="1235" height="1042" alt="Image" src="https://github.com/user-attachments/assets/2e7b0f98-9de7-4ae6-8585-62c247195caf" />

**Comment by merriemcgaw** [comment]
@LeafShi1 no - this should not need to be backported if we've already fixed it in .NET 9. Can you please check to see if we have an issue targeting .NET Framework in our AzDO instance? If so, @shindepra can close this particular issue as we'll be tracking the .NET Framework side internally. Note that the SQL team may have to obtain exceptions in this situation and to work with me for any questions.

**Comment by LeafShi1** [comment]
There are no active issues in .NET Framework internal AzDO instance, so an internal bug https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2542782 has been filed to track this issue

**Comment by merriemcgaw** [comment]
@shindepra can we close this issue  now?

**Comment by kupatkar99** [comment]
Closing the bug and this issue will be tracked in https://github.com/dotnet/winforms/issues/13995
#Closed;
